src/ScriptCs.Core/ScriptLibraryWrapper.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ScriptCs.Contracts;
+
+namespace ScriptCs
+{
+    public abstract class ScriptLibraryWrapper
```

Comment:
One caveat here is this requires the MonoHost to have an explicit implementation of Env as it already has a public static property. I added this and it compiles fine.